### DISCUSSION TITLE
Microsoft Store runFullTrust warning

### DISCUSSION THREAD

**Discussion by DonaldBarron** [unoplatform/uno#10191] [Q&A]
Im not sure if this is the right place for this question so if not please direct me to the correct place. Using Visual studio 2022 17.3.6 and Uno platform templates 4.5.12.8 all my submitted .msix packages seem to be flagged in the store as being full trust even though I only 
 have internet(client)selected in the capabilities.

I even tried creating a brand new project with a brand new app in the store made no other changes(apart from the annoying bits you have to fill in to submit to the store ) and left all the capabilities unticked and it still seems to come up with this warning.

I tried the same with a plain(non uno) UWP app and didnt get any of those issues.

Is there anyway to change this? Am I doing something wrong?

**Comment by pkar70** [comment]
As far as I know msix packaging adds fulltrust, and is designed as a Store wrapper for non-UWP apps. "standard" UWP app has another package format, and uses only  permissions defined in manifest.

**Reply by DonaldBarron** [reply]
Ah ok, I was not aware of this. Thank you that gives me a line of enquiry to follow. Is Microsoft idea then that anyone who tries to move on to win ui 3 will be publishing everything as full trust?

**Comment by pkar70** [comment]
I think that answer is YES.
It seems like WinUI3 is something like replacement for standard Windows app, and standard Windows app always has full access. MFC, WPF, etc. UWP app is running in a sandbox (sort of), but WinUI app - not.

**Comment by DonaldBarron** [comment]
Thank you pkar70 you helping point me in the next correct direction.

So I found a post about setting your winui3 app to run as partial trust. I created a "plain" winui3 project just to test it and the option certainly appears in my test. (note the image is from the article rather than my test but they show the same thing).  Where under in my test it was under Dependencies>Applications>YourAppName and the properties at this level let you switch from full trust to partial trust.  
![image](https://user-images.githubusercontent.com/107513594/199059190-656cbce8-346a-45d2-b084-02a2f8a7adb6.png)
I had a look in  Uno projects and couldnt Find this option. (I did try manually removing the 
    <rescap:Capability Name="runFullTrust" /> section from the AppManifest.xml but it doesn't seem to change the output msix files).

I think my question is do you happen to know where i should got look when using uno  to find this option?

The post I was referring to:
https://nicksnettravels.builttoroam.com/winui-appcontainer/
